### PR TITLE
LPS-147824 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-147824

NOTE: This test needs to assert the future event, so we have to add a "Pause" to wait for the entry to be displayed after setting the blog display date. And the wait time has reached the minimum value that it can be set.

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
@Jeremy-Ch  Putting on hold until I can confirm there is absolutely no way right now to work around pauses

File: portal-web/test/functional/com/liferay/portalweb/macros/json/blogs/JSONBlog.macro [hunk 1]
```diff
@@ -324,6 +324,39 @@ definition {
 		JSONCurlUtil.delete("${curl}");
 	}
 
+	@summary = "This calls the headless API to subscibe on a blog widget"
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/blogs/JSONBlog.macro:327]: subscribe

File: portal-web/test/functional/com/liferay/portalweb/macros/json/blogs/JSONBlog.macro [hunk 2]
```diff
@@ -324,6 +324,39 @@ definition {
...
+	macro subscribeOnBlogsWidget {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/blogs/JSONBlog.macro:328]: subscribeTo

**Comment by Jeremy-Ch** [comment]
@austinchiang I will update and resend it, thanks.

**Comment by Jeremy-Ch** [comment]
@austinchiang I will update and resend it, thanks.

**Comment by Jeremy-Ch** [comment]
See https://github.com/austinchiang/liferay-portal/pull/1308

**Pull request closed without merging**